suite-native/analytics/src/analytics.ts

```diff
@@ -11,5 +11,10 @@ export const analytics = new Analytics<SuiteNativeAnalyticsEvent>({
 
 if (isDebugEnv()) {
     // Do not send analytics in development
-    analytics.report = () => {};
+    analytics.report = (event: Event) => {
+        if (process.env.EXPO_PUBLIC_IS_ANALYTICS_LOGGER_ENABLED) {
```

Comment:
i this condition (and env variable) needed when we only log this while `isDebugEnv()` ?

Reply:
Yes, I think it is still useful - mostly you don't care about analytics during development and you don't want console spammed by it...